### ISSUE TITLE
[AppVeyor] cannot open file libpq.lib

### ISSUE THREAD

**Issue by tt4g** [tt4g/conan-soci#18]
```
"C:\Users\appveyor\.conan\data\SOCI\4.0.0\user\channel\build\a7bd270976e92034c32d578a48ec7f9c2e8b71c4\src\backends\postgresql\soci_postgresql.vcxproj" (default target) (9) ->
       (Link target) -> 
         LINK : fatal error LNK1104: cannot open file 'C:\Users\appveyor\.conan\data\libpq\9.6.9\bincrafters\stable\package\a4cf40bc51726f42f50629891b31dc99d118ebdc\lib\libpq.lib' [C:\Users\appveyor\.conan\data\SOCI\4.0.0\user\channel\build\a7bd270976e92034c32d578a48ec7f9c2e8b71c4\src\backends\postgresql\soci_postgresql.vcxproj]
```

**Comment by tt4g** [comment]
Apparently the cause seems to be in the compiler flag /MTd.
libpq build successful with /MTd, but link error when SOCI build.

/MT and /MD have been successfully built.

**Comment by tt4g** [comment]
Fixes 8577c7a75155ab0b24a5eaf05ee044f1bdada7d2
